### PR TITLE
We should process formaction attribute (close #988)

### LINKED ISSUE #988: We should process "formaction" attribute

### PR DESCRIPTION
**Pull request by churkin**
/cc @LavrovArtem @miherlosev

### CODE REVIEW

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
FPR

File: test/client/fixtures/sandbox/node/dom-processor-test.js [hunk 1]
```diff
@@ -22,8 +23,8 @@ QUnit.testDone(function () {
 });
 
 test('iframe', function () {
-    var iframe         = nativeMethods.createElement.call(document, 'iframe');
-    var storedAttrName = domProcessor.getStoredAttrName('sandbox');
+    var iframe                             = nativeMethods.createElement.call(document, 'iframe');
+    var storedAttrName                     = domProcessor.getStoredAttrName('sandbox');
```
Comment (LavrovArtem) [inline test/client/fixtures/sandbox/node/dom-processor-test.js]: extra spaces

File: test/client/fixtures/sandbox/node/dom-processor-test.js [hunk 2]
```diff
@@ -105,7 +106,7 @@ test('script text', function () {
     var processedScript = processScript(script, true);
 
     nativeMethods.appendChild.call(document.body, div);
-    div.innerHTML = '\<script\>' + script + '\</script\>';
+    div.innerHTML       = '\<script\>' + script + '\</script\>';
```
Comment (LavrovArtem) [inline test/client/fixtures/sandbox/node/dom-processor-test.js]: extra spaces

File: test/client/fixtures/sandbox/node/attributes-test.js [hunk 3]
```diff
@@ -27,9 +27,9 @@ QUnit.testDone(function () {
 // NOTE: IE11 has a strange bug that does not allow this test to pass
 if (!browserUtils.isIE || browserUtils.version !== 11) {
     test('onsubmit', function () {
-        var etalon = nativeMethods.createElement.call(document, 'form');
-        var form   = document.createElement('form');
-        var check  = function () {
+        var etalon      = nativeMethods.createElement.call(document, 'form');
+        var form        = document.createElement('form');
+        var check       = function () {
```
Comment (LavrovArtem) [inline test/client/fixtures/sandbox/node/attributes-test.js]: extra spaces

File: test/client/fixtures/sandbox/node/dom-processor-test.js (hunk 1 above)
Comment (churkin) [inline test/client/fixtures/sandbox/node/dom-processor-test.js]: It is WebStorm formatter problem

File: test/client/fixtures/sandbox/node/dom-processor-test.js (hunk 2 above)
Comment (churkin) [inline test/client/fixtures/sandbox/node/dom-processor-test.js]: It is WebStorm formatter problem

File: test/client/fixtures/sandbox/node/attributes-test.js (hunk 3 above)
Comment (churkin) [inline test/client/fixtures/sandbox/node/attributes-test.js]: It is WebStorm formatter problem

**Pull request merged**